zerver/views/presence.py

```diff
@@ -54,13 +54,27 @@ def get_presence_backend(request: HttpRequest, user_profile: UserProfile,
 @has_request_variables
 def update_user_status_backend(request: HttpRequest,
                                user_profile: UserProfile,
-                               away: bool=REQ(validator=check_bool),
+                               away: Optional[bool]=REQ(validator=check_bool, default=None),
+                               info: Optional[str]=REQ(default=None),
```

Comment:
Let's call this `status` or `status_message` -- I think `info` is a bit too vague.